tests/kernel/mbox/mbox_api/boards/qemu_x86_lakemont.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Need a little bit more stack space
+# after moving to new ztest API
+CONFIG_TEST_EXTRA_STACK_SIZE=64
```

Comment:
Might consider just a slightly larger increment.  It's true that our test suite really likes small stacks, but if 64 more bytes is enough, then we're within 64 bytes of a known overflow on at least one platform!  Someone is going to have to come back to this test in the near future and adjust it again, it's almost guaranteed.